de.fu_berlin.inf.dpp.server/src/de/fu_berlin/inf/dpp/server/filesystem/ServerResourceImpl.java

```diff
@@ -16,18 +16,22 @@
 public abstract class ServerResourceImpl implements IResource {
 
     private IWorkspace workspace;
+    private IProject project;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Rename field "project" to prevent any misunderstanding/clash with field "PROJECT" defined in interface "de.fu_berlin.inf.dpp.filesystem.IResource". ([squid:S1845](https://sonarcloud.io/organizations/saros/rules#rule_key=squid%3AS1845))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=saros_complete&pullRequest=141&issues=AWVmay6FelKyZq-pzYE_&open=AWVmay6FelKyZq-pzYE_)

[ISSUE_KEY]: AWVmay6FelKyZq-pzYE_